test/e2e/network/conntrack.go

```diff
@@ -658,4 +658,90 @@ var _ = common.SIGDescribe("Conntrack", func() {
 		framework.Logf("boom-server pod logs: %s", logs)
 		framework.Logf("boom-server OK: did not receive any RST packet")
 	})
+
+	// This test checks that conntrack entries for old target ports are cleaned up when a UDP Service’s target port changes.
+	// It also covers cases where it prints the serverport entries, to make sure cleanup works correctly.
+	// This verifies the fix in #130542.
```

Comment:
`// This verifies the fix in https://pr.k8s.io/130542`

we've got custom URL shorteners for PRs and issues so you can refer to them easily from the code